share/spack/gitlab/cloud_pipelines/stacks/aws-pcluster-neoverse_v1/spack.yaml

```diff
@@ -1,105 +1,93 @@
 spack:
   view: false
-
   definitions:
-  - apps:
-    - gromacs
-    - mpas-model
-    - mpich
-    - openfoam
-    - quantum-espresso
-    - wrf
-
-  - targets:
-      - 'target=neoverse_v1'
-      - 'target=neoverse_n1'
-
+    - apps:
```

Comment:
Can you use spack's own yaml format please (e.g. `spack -e . add zlib`) and remove it again.